roadmap/implementers-guide/src/node/backing/statement-distribution.md

```diff
@@ -1,107 +1,453 @@
-# Statement Distribution
-
-The Statement Distribution Subsystem is responsible for distributing statements about seconded candidates between validators.
+# Statement Distribution (Draft)
+
+This is a first draft for the implementer's guide, which I wrote as a dump of
+all info and conclusions I could find, but in one place. This helped me a lot to
+connect all the "dots", but it may be too much info. I need feedback on the
+following:
+
+-   [ ] Where it is too technical, I can move stuff into mod-docs.
+-   [ ] And of course, accuracy/thoroughness.
+
+There is some redundancy, but I came across a saying I liked today: "DRY is for
+code, not docs." Re-stating the same thing multiple times helped with my
+comprehension. But feedback is welcome.
+
+Also, I will convert it to prose once we know what info is staying. The bullets just helped me stay organized.
+
+## Overview
+
+-   ****Goal:**** every well-connected node is aware of every next potential parachain
+    block.
+-   Validators can either:
+    -   receive parachain block from collator, check block, and gossip statement.
+    -   receive statements from other validators, check block, gossip forward
+        statement if valid.
+-   Why:
+    -   Validators must have statements from all (?) other validators
```

Comment:
Not only statements, but the candidates and persisted validation itself. Any validator can be selected as the next relay-chain block author, and this is not revealed in advance for security reasons. As a result, all validators must have a up to date view of all possible parachain candidates + backing statements that could be placed on-chain in the next block.

Reply:
Validators need to consider _all_ seconded candidates within their own group, because that's what they're assigned to work on. Validators only need to consider backable candidates from other groups.